subsys/logging/log_output.c

```diff
@@ -250,7 +252,7 @@ static void std_print(struct log_msg *msg,
 }
```

Comment:
Please add default: and __ASSERT() catching unsupported number of arguments.

Reply:
added